test/TestServerProjects/body-number/Generated/Operations/NumberOperations.cs

```diff
@@ -512,7 +512,7 @@ public static async ValueTask<Response> PutBigDecimalPositiveDecimalAsync(Client
                 request.Headers.Add("Content-Type", "application/json");
                 using var content = new Utf8JsonRequestContent();
                 var writer = content.JsonWriter;
-                writer.WriteNumberValue(99999999.99);
+                writer.WriteNumberValue(99999999.99M);
```

Comment:
And what caused this?

Reply:
using the correct Literal implementation instead of object.ToString